apps/ai-image-tagging/lambda/serverless.yml

```diff
@@ -35,10 +44,30 @@ package:
 functions:
   app:
     handler: index.handler
+    environment:
+      TABLE_NAME: ${self:custom.tableName}
     events:
       - http:
           path: /
           method: ANY
       - http:
           path: /{proxy+}
           method: ANY
+
+resources:
+  Resources:
+    UsageTable:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: ${self:custom.tableName}
+        AttributeDefinitions:
```

Comment:
For a time series, I'd suggest we have the spaceId as the primary key and the period as a number represented as a timestamp truncated to the series granularity (see for instance https://momentjs.com/docs/#/manipulating/start-of/)

Reply:
What the benefits would be? I think having it as it is will result in the easiest queries.

Reply:
The timestamp representation is used for storage efficiency and access performance reasons - especially for slices of data. Also we can use standard functions and don't have to do manual date rendering which often leads to nasty bugs. Spend too much time of my life chasing them down.